tests/src/KeyManagerTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * Provides \Drupal\Tests\key\KeyManagerTest.php
+ */
+
+namespace Drupal\Tests\key;
+
+use Drupal\key\Plugin\KeyType\SimpleKey;
+use Drupal\key\Entity\Key;
+use Drupal\key\KeyManager;
+
+class KeyManagerTest extends KeyTestBase {
```

Comment:
Do you know we also have src/Tests for SimpleTests?

This seems like a parallel but related set of tests. Can we somehow merge them and get the same results?